### ISSUE TITLE
Displaying the relevant picture for each word on the screen

### ISSUE THREAD

**Issue by hasantezcan** [batin/fastReadingTool#2]
We can put a display of relevant pictures for each word while the word changing on the screen.

**Comment by aykutkardas** [comment]
> We can put a display of relevant pictures for each word while the word changing on the screen.

This would be a very useful suggestion if the goal was language learning. But I think this would be useless for speed reading.

**Comment by hasantezcan** [comment]
Yes, you are right this feature is a little bit out of scope in this project.  We found this while making brainstorming and we thought that feature can teach new words to kids. As you thought we can use it for language learning. Maybe we can add those features as an extra mode.
